src/pagination/pagination.component.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.pagination.component:pfPagination
+ * @restrict E
+ *
+ * @param {number} pageNumber The current page number
+ * @param {number} numTotalItems The total number of items in the data set.  When a filter is applied, update the <code>numTotalItems</code>
+ * accordingly.
+ * @param {number[]} pageSizeIncrements (optional) Page size increments for the 'per page' dropdown.  If not
```

Comment:
I don't see the harm in providing this info., just to let them know we're expecting an array of numbers.  I can trim it to just 'Array' if you feel that's better.